src/index.html

```diff
@@ -2,16 +2,178 @@
+            id="birth-date-input"
+          >
+        </div>
+        <div class="form-field">
+          <label for="agreement-input">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement-input"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label>E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+
+        </div>
+        <div class="form-field">
+          <label>Password:
+            <input
+              type="password"
+              name="name"
+              minlength="4"
+              maxlength="20"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="y"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              value="#000000"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time-to-bed"
+              step="1"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              min="0"
+              max="10"
+              value="0"
+            >
+            </input>
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label for="surname-input">Comments:
+            <textarea
+              name="comments"
+              cols="20"
+              rows="2"
+            >
+            </textarea>
```

Comment:
```suggestion
            <textarea
              name="comments"
              cols="20"
              rows="2"
          ></textarea>
```

it should fix extra spaces in textarea field

Reply:
Sorry, and i've some question. Should I used case 1 always? Or is it possible to use the case 2 ?
![image](https://user-images.githubusercontent.com/62157487/222900348-e160f2d4-4212-4dcf-9d55-a8cf3fca5988.png)

Reply:
No, you can also you Case 2, it depends on line length and number of attribues

Reply:
In examples above you can use Case 2